meta/runtime.yml

```diff
@@ -1,3 +1,3 @@
 ---
-requires_ansible: '>=2.9.6'
+requires_ansible: '>=2.11'
```

Comment:
Is this really the right thing to do? I don't think we really do require that later version of Ansible.

In another collection I've just turned off the `meta-unsupported-ansible` rule as it doesn't really make sense here.